tests/check_docs.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess
+
+root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+def main():
+  build_output = os.path.join(root, 'site', 'build', 'text', 'docs', 'tools_reference', 'emcc.txt')
+  docs_file = os.path.join(root, 'docs', 'emcc.txt')
+
+  if not os.path.exists(build_output):
+    print('doc build output not found: %s' % build_output)
+    return 1
+
+  with open(build_output, 'r') as f:
+    emcc_docs_output = f.read()
+
+  with open(docs_file, 'r') as f:
+    emcc_docs = f.read()
+
+  if emcc_docs_output != emcc_docs:
+    print('contends of checked in docs/emcc.txt does not match build output:')
+    subprocess.call(['diff', '-u', build_output, docs_file])
```

Comment:
does this print out the diff, so that a user can easily apply it locally instead of installing sphinx etc.?

Reply:
No.. is just for debugging purposes.

Reply:
We do that on binaryen with clang-format, I think it's useful. Basically just print the diff if there is one, so contributors can just apply it.

Reply:
Sure I mean this does print the diff.  I just wasn't my intention that you would apply it.  But you could.